### ISSUE TITLE
Killing Spree causing too few attacks

### ISSUE THREAD

**Issue by fendwe** [chromiecraft/chromiecraft#3492]
### What client do you play on?

enUS

### Faction

- [ ] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59
- [X] 60

### Current Behaviour

Using the ability [Killing Spree](https://wowgaming.altervista.org/aowow/?spell=51690) results in only 4 attacks being made, instead of 5.
Hence the 8 hits.
![image](https://user-images.githubusercontent.com/61170874/167836189-9085180b-0131-479c-abfd-74f4cc22ca99.png)

### Expected Blizzlike Behaviour

Killing Spree should make the Rogue attack 5 times.
![image](https://user-images.githubusercontent.com/61170874/167836643-bf738958-864e-472f-8541-c5adf0424ab5.png)

### Source

Source is the ability tooltip.

### Steps to reproduce the problem

1. Use Rogue.
2. cast Killing Spree on single target
3. observe amount of hits

### Extra Notes

The problem was reportd before on AC, but i guess it was dropped?
See https://github.com/azerothcore/azerothcore-wotlk/issues/10955

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/c7f74e563758241e29721e95b86aed0093a8856a

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

**Comment by Gultask** [comment]
Hi, fendwe, thanks for the report!

I was unable to reproduce in either a local AC, the PTR and even the Live CC realm. But since the fix was fairly recent, then it's very possible that Chromiecraft hadn't updated yet.
Chromiecraft was updated 8 hours ago, could you see if it still happens?

**Comment by fendwe** [comment]
You are right, for some reason skada is just not picking up on all the hits.
I'm closing this.
